docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,118 @@
+# Zowe certificates questionnaire
+
+Answering [the questions](#certificates-configuration-questionnaire) in this questionnaire will help to make your choice of certificates setup according to your planned Zowe deployment.
+
+If you need to learn more about the certificates based security terminology, review the [certificates concepts](../appendix/zowe-security-glossary#certificate-concepts) in our Security Glossary.   
+
+If you want to lean more about how Zowe uses digital certificates, visit the [Zowe Certificates Overview](../getting-started/zowe-certificates-overview).
+
+Before you answer [the questions](#certificates-configuration-questionnaire) you could also review the [Certificate Configuration Scenarios](certificate-configuration-scenarios.md) to understand better the certificates configuration options. 
+
+## Certificates configuration options
+
+1. Certificates storage type.
```

Comment:
The phrases in 1 and 2 should not have periods b/c they are not complete sentences.

Reply:
Done.